### ISSUE TITLE
if resourcePackage is not defined all packages should be included

### ISSUE THREAD

**Issue by fenoloco** [swagger-api/swagger-core#1098]
It's to hard to list all the packages, if someone needs map all the packages it easy to leave it as default.

The idea is something like this.

```java

 if(resourcePackage != "") {
      String[] parts = resourcePackage.split(",");
      for(String pkg : parts) {
        if(!"".equals(pkg)) {
          acceptablePackages.add(pkg);
          config.addUrls(ClasspathHelper.forPackage(pkg));
        }
      }
    }
//add this part
else{
 allPackageAllow = true;
}

...................

//and here add an or

 for(Class<?> cls : classes) {
      if(allPackageAllow || acceptablePackages.contains(cls.getPackage().getName()))
        output.add(cls);
    }

```

Regards,
Mauricio

**Comment by webron** [comment]
This can also be resolved by #1011. However, it's a nice idea.

**Comment by fehguy** [comment]
closing as #1011 supports this (just scan *all* packages)
